core/table.ts

```diff
@@ -153,6 +153,39 @@ export interface IBigQueryOptions {
 const IBigQueryOptionsProperties = () =>
   strictKeysOf<IBigQueryOptions>()(["partitionBy", "clusterBy", "updatePartitionFilter", "labels"]);
 
+/**
+ * Options for creating tables within Presto projects.
+ */
+export interface IPrestoOptions {
+  /**
+   * The key with which to partition the table. Typically the name of a timestamp or date column.
+   *
+   * For more information, read the partiton by docs for the connection you are using.
+   */
+  partitionBy?: string[];
+
+  /**
+   * The key with which to partition the table. Typically the name of a timestamp or date column.
+   *
+   * For more information, read the partiton by docs for the connection you are using.
+   */
+
+  /**
+   * If catalog and schema are specified in the request, then leading prefixes
+   *
+   * can be ignored, e.g. "catalog.schema.table" can be referred to as just "table".
+   */
+  catalog?: string;
```

Comment:
i don't think we should have these - let's align `database` (part of any action `config` block) with `catalog`, and we already have `schema` there too.